### DISCUSSION TITLE
Reload issue in TeamCity(headless mode)

### DISCUSSION THREAD

**Discussion by P-Neha** [getgauge/taiko#2040] [Development]
I'm trying to reload my website if the contents of the website don't load properly on the first instance. The reload action works fine when the headless mode is set to false. In teamcity I run the script in headless mode and I'm seeing this below issue. Could someone please help me on this?

Code:
 if(await text("Log In").exists() == false){
            await reload(process.env.LOGIN_URL);
            await waitFor(text("Log In"));   
        } 

Error in headless mode if reload occurs:
 Error: socket hang up
[15:14:07][Step 3/4] at connResetException (node:internal/errors:642:14)
[15:14:07][Step 3/4] at Socket.socketCloseListener (node:_http_client:435:25)
[15:14:07][Step 3/4] at Socket.emit (node:events:390:22)
[15:14:07][Step 3/4] at TCP.<anonymous> (node:net:665:12) {
[15:14:07][Step 3/4] code: 'ECONNRESET'
[15:14:07][Step 3/4] }

**Comment by NivedhaSenthil** [comment]
can you please provide us with more info regarding the environment(os, node version, taiko version) to analyse the issue, also is this issue happening locally as well when run on headless mode?

**Reply by P-Neha** [reply]
Yes it happens locally as well when I run in headless mode. Here are my local details
OS - Windows
Node Version - v14.15.1
Taiko Version - 1.2.5

**Reply by NivedhaSenthil** [reply]
no luck trying to reproduce the issue..will be helpful if you can provide us with a sample to replicate meanwhile

**Reply by P-Neha** [reply]
That might need access to my company's website which only works on vpn. Is there any other way we could try?
